packages/core/src/textures/TextureGCSystem.ts

```diff
@@ -139,7 +139,7 @@ export class TextureGCSystem extends System
         const tm = this.renderer.texture;
 
         // only destroy non generated textures
-        if (displayObject._texture && displayObject._texture._glRenderTargets)
+        if (displayObject._texture?._glRenderTargets)
```

Comment:
Anyone know what the heck is `_glRenderTargets`, I couldn't find a reference to it anywhere in the code. @GoodBoyDigital or @ivan?

Reply:
displayObject._texture?.framebuffer

Reply:
:+1:

Reply:
_glRenderTargets <-- is super old check to see if the texture was render texture. but got missed in a past refactor